services/ui-src/src/components/cards/EntityCard/EntityCardTopSection.tsx

```diff
@@ -0,0 +1,57 @@
+// components
+import { Text } from "@chakra-ui/react";
+// utils
+import { AnyObject, ModalDrawerEntityTypes } from "types";
+
+export const EntityCardTopSection = ({
+  entityType,
+  formattedEntityData,
+}: Props) => {
+  switch (entityType) {
+    case ModalDrawerEntityTypes.ACCESS_MEASURES:
+      return (
+        <>
+          <Text sx={sx.description}>
+            {formattedEntityData.standardDescription}
+          </Text>
+          <Text sx={sx.subtitle}>General category</Text>
+          <Text sx={sx.subtext}>{formattedEntityData.standardType}</Text>
+        </>
+      );
+    case ModalDrawerEntityTypes.SANCTIONS:
+      return (
+        <>
+          <Text sx={sx.description}>Sanctions TODO</Text>
+        </>
+      );
+    case ModalDrawerEntityTypes.QUALITY_MEASURES:
+      return (
+        <>
+          <Text sx={sx.description}>Quality Measures TODO</Text>
+        </>
+      );
```

Comment:
Nit: Maybe I'm wrong, but since these are both one liners can they return just the Text component?

`return <Text sx={sx.description}>Sanctions TODO</Text>`

Reply:
good callout! Fixed!